conf/db/migration/metrics_portal_ddl/common/V3__create_alerts_table.sql

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2015 Groupon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CREATE TABLE IF NOT EXISTS portal.alerts (
+    id UUID PRIMARY KEY,
+    version INTEGER NOT NULL DEFAULT 1,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now(),
+    name VARCHAR(255) NOT NULL,
+    cluster VARCHAR(255),
+    service VARCHAR(255) NOT NULL,
+    context VARCHAR(255) NOT NULL,
```

Comment:
The only concern I see with using a pg enum here is that when we change the enum in code, it will incur an additional migration and the developer will need to remember that but I can add a comment in the Context model for this. That should help.